### ISSUE TITLE
New work record for

### ISSUE THREAD

**Issue by DariaElagina** [BetaMasaheft/Documentation#1782]
BL Orient 743 (not online), 
has following texts which I can not identify:
1) A hymn (?): እደግም፡ በረድኤተ፡ እግዚአብሔር፡ ወራእየ፡ መንክረ፡ ዘተመሰለ፡ ሎሙ፡ ለነብያት፡ ወኵሎሙ፡ በበአስማቲሆሙ፡
                              ወርእዩ፡ ትእምርተ፡ አምላኮሙ፡ ዘኮነ፡ ወዘይከውን፡ እነግር፡ ወእብል፡ እሳትኒ፡ ዘኮነ፡ በቅድመ፡ ሙሴ፡ ወመብረቅኒ፡ ዘኮነ፡
                              በቅድመ፡ ዕዝራ። ወብእሴ፡ አረጋዌ፡ ዘሄኖክ፡ ዘኮነ፡ ወወረደ፡ ነቅዐ፡ ጽድቅ፡ እምሰማይ፡ ወአዕረፈ፡ ጸጋ፡ ለከዊነ፡
                              ትእምርት፡ ላዕሌሁ።
2) The Song of Songs, concerning the Son and the Christian Church, and His Mother, an exposition of words and phrases in the Song of Songs: 
በስመ፡ አብ፡ መኀልየ፡ መኃልይ፡ በእንተ፡ ወልድ፡ ወበእንተ፡ ቤተ፡ ክርስቲያን፡ ወበእንተ፡ እሙ፡ ሰላሙ፡
                            ለገብርኤል፡ አጥባቲሃ፡ ዘይቤ፡ ዕፍረት። ሃይማኖት፡ አፈው፡ አፈወ፡ ክርስቲያን፡ ስመ፡ አብ፡ ወወልድ፡
                            ወመንፈስ፡ ቅዱስ፡ ደናግል፡ መላእክት፡ ድኅሬሁ፡ ዕርገቱ፡ ጽርኁ፡ ዘይቤ፡ ሰማያት። ለንጉሥ፡ ዘይቤ፡
                            ወልድ። ንትፌሣሕ፡ ወንትኃሠይ፡ ብከ፡ ዘይቤ፡ በእንተ፡ ትንሣኤ። ናፍቅር፡ አጥባትኪ፡ ዘይቤ፡ ሥጋሁ፡
                            ወደሙ። ርቱዕ፡ አፍቅሮቱ፡ ዘይቤ፡ አሚን።
3) Expositions of some passages of the Old and New Testament:
ኦሪት፡ ዘፀአት። ወበዕለተ፡ ዓርብ፡ <sic>ያተጋብኡ፡</sic> ካዕበተ፡ ጎሞር፡ ወቦአ፡ ኵሉ፡ መኳንንተ፡ ማኅበር።

Any ideas on the texts? Should I just create entries for them?
@thea-m  @DenisNosnitsin1970

**Comment by DariaElagina** [comment]
Since I was able to find some additional information, the issue is temporarily closed.

**Comment by DenisNosnitsin1970** [comment]
Yes, the statement on p. 282b is unclear, but i see that it may be just an imperfection in wording from whomever it could originate. I assume that it originates from the author because I see a very close wording - and the same problem - in French though, on p. 32 of Van Donzel's edition of "Anqasa amin" (here the text is complex as it presents an opinion by another person, L. Ricci. However, I assume that the author had in mind somehow not the entire contentof the bothe manuscripts, but maybe a part, i.e. the notes concerning the translator (that both are similar?). Is it possible? (perhaps the part "in ms. BN Abbadie 9 ... 743" can be just disregarded).

**Comment by DariaElagina** [comment]
Thank you, Denis! I did not even come to reading this statement like that. I think you are right So, the only question is, if the work records for those texts should be created.

**Comment by eu-genia** [comment]
is there a solution in the meanwhile?

**Comment by thea-m** [comment]
https://github.com/BetaMasaheft/Documentation/issues/2107
